test_data/rules/cascade_invocations.dart

```diff
@@ -57,13 +57,15 @@ class Foo {
 }
 
 void noCascadeWithGetter() {
-  final foo = new Foo();
+  final foo = Foo();
   foo.baz(); // LINT
   foo.bar; // LINT
   foo.foo(); // LINT
   foo.bar = 8; // LINT
+  // ignore: invalid_null_aware_operator
```

Comment:
It isn't clear to me that it's useful to test the behavior of the lint in code with compilation errors. When it was valid to use the null aware access operator (pre null-safety) it made sense to me, but now there's already a different diagnostic, and I don't think we generally attempt to suppress lints in those cases (though maybe we should).

Reply:
I agree 100%.  This was a mistake on my part.  I don't think this is a useful state to be testing and in any event in the case where it is, I'd like to break those out into separate unit tests rather than suppressing w/ ignores.

Thanks for the catch!